### ISSUE TITLE
Automatically press enter to input the transcribed text.

### ISSUE THREAD

**Issue by Wuzzooy** [Sharrnah/whispering-ui#15]
Hello,
Thank you for this massive work.

Is it possible to add an option with the keyboardTyping plugin to automatically press enter once the transcribed text from speech to text is done so we don't have to press manually enter key ?

It could be done by setting a time when there is silence or when using a key push to talk, it could be done by releasing the key.

Thank you.

**Comment by Sharrnah** [comment]
Thank you for the suggestion.

I just added a "in_game_chat" settings dictionary to the Keyboard typing plugin (now version 1.0.4).

This has two values. one key that is pressed before the text is typed, and one after the text is typed.
its disabled by default. The post and pre typing keys are what i think are the most common hotkeys for in-game chats.

So if enabled, it presses "T" before typing,  and Enter after it finished typing.

I hope thats what you where looking for.

![grafik](https://github.com/Sharrnah/whispering-ui/assets/55756126/9bb89d53-0a23-4f73-ac5d-0f9c95a911fe)

let me know if that works.

**Comment by Wuzzooy** [comment]
Thank you it's even better than what i asked with the pre typing key function and yes it does work.

**Comment by Sharrnah** [comment]
NIce. :) Thanks for confirming it works.

WIll close this then.
Feel free to open another issue for any suggestion or question.
